ImageLounge/src/DkCore/DkSettings.cpp

```diff
@@ -600,7 +602,7 @@ void DkSettings::save(QSettings &settings, bool force)
     // always save (user setting)
     settings.setValue("defaultJpgQuality", app_p.defaultJpgQuality);
     settings.setValue("appMode", app_p.appMode);
-    settings.setValue("currentAppMode", app_p.currentAppMode);
+    // settings.setValue("currentAppMode", app_p.currentAppMode);

```

Comment:
Should we remove this as `currentAppMode` is replaced?

Reply:
See my next comment